package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@vtex/phone",
+  "name": "front.phone",
```

Comment:
I needed to rename the package because when pulling the files from io.vtex.com it does not behave correctly when the application name has a forward-slash, and returned a 404 error for every request

Reply:
@lucasecdb newer versions of this package won't be published on npm, will they?

Reply:
sadly no :-/

but the CI isn't working for quite some time, so we weren't publishing them anymore (it is related to a migration from travis-ci.org to travis-ci.com and we didn't update the integration in GH)

Reply:
:okay: